pyi.py

```diff
@@ -2129,5 +2138,9 @@ def parse_options(options: argparse.Namespace) -> None:
     '"a tuple of length 1, in which the sole element is of type {typ!r}". '
     'Perhaps you meant "{new}"?'
 )
+Y091 = (
+    'Y091 "Callable" in argument annotations '
+    'should generally have "object" for the second parameter, not "{bad_return}"'
```

Comment:
"second parameter" is quite confusing because I would assume it means the second parameter to the callable, not the second type parameter to `Callable`. I think this is clearer:

```suggestion
    'should generally have "object" as the return type, not "{bad_return}"'
```